### ISSUE TITLE
Getting warning about triggerEvent and propsFromTrigger

### ISSUE THREAD

**Issue by giacomoalonzi** [fkhadra/react-contexify#187]
**Do you want to request a *feature* or report a *bug*?**
Bug

**What is the current behavior?**

I'm getting "React does not recognize the `triggerEvent` prop on a DOM element." or  react_devtools_backend.js:2557 Warning: React does not recognize the `propsFromTrigger` prop on a DOM element.

**What is the expected behavior?**
No warnings

**Which versions of React, and which browser / OS are affected by this issue? Did this work in previous versions of React?**
"react": "17.0.2". 
Never used other versions

**Comment by giacomoalonzi** [comment]
Warning: React does not recognize the `triggerEvent` prop on a DOM element. If you intentionally want it to appear in the DOM as a custom attribute, spell it as lowercase `triggerevent` instead. If you accidentally passed it from a parent component, remove it from the DOM element.

Warning: React does not recognize the `propsFromTrigger` prop on a DOM element. If you intentionally want it to appear in the DOM as a custom attribute, spell it as lowercase `propsfromtrigger` instead. If you accidentally passed it from a parent component, remove it from the DOM element.

any help?

**Comment by fkhadra** [comment]
without a reproduction I cannot help :(

**Comment by jackzang98** [comment]
Hello. Did you solve this problem, I also encountered the same problem

**Comment by bardaxx** [comment]
> Hello. Did you solve this problem, I also encountered the same problem

Nope, used this lib on another project and I got the same issue :\

**Comment by mrdulin** [comment]
Same issue. It seems the warning thrown from `<Menu/>` component [here](https://github.com/fkhadra/react-contexify/blob/8d9fc63ac13040d3250e8eefd593d50a3ebdd1e6/src/components/Menu.tsx#L296)

**Comment by MattiHusuTrimble** [comment]
This happens to me also in: 

Windows 10
react 18.2.0
"react-contexify": "^6.0.0"
Chrome version 120.0.6099.225

**Comment by char-m** [comment]
This happens to me also in

- win 10
- react 18.2.0
-  react-contexify": "^6.0.0

This happens when `useEffect` is used to set the content of the menu. 

EDIT: I solved this by making sure the content is inside Item tags. I for example added a spinner component inside div to menu and when ready the actual items wrapped inside `<div></div>`. This works but it produces this error. So I had like this:

```
export function ContextMenu(props: ContextMenuProps) {
  const [content, setContent] = useState<ReactNode | undefined>(undefined);
  const someStatus = useSelector(selectSomeStatus);
  useEffect(() => {
    if (someStatus === 'pending') {
      setContent(
        <div className='d-flex justify-content-center'><SomeSpinner/></div>);
    } else {
      const items = getMenuItems().
        map(item => <Item key={item.key} onClick={() => item.action()}>{item.key}</Item>);
      setContent(<div>{items}</div>);
    }
  }, [someStatus]);

  return (
    <Menu id={props.menuId}>
      {content}
    </Menu>
  );
}
```

When I removed the divs in BOTH cases I got rid of the error:

```
  useEffect(() => {
    if (someStatus === 'pending') {
      setContent(
        <Item className='d-flex justify-content-center'><SomeSpinner/></Item>);
    } else {
      const items = getMenuItems().
        map(item => <Item key={item.key} onClick={() => item.action()}>{item.key}</Item>);
      setContent(items);
    }
  }, [someStatus]);
```

**Comment by ROTGP** [comment]
Just wrap your content in empty tags like so:  <>your content...</>

**Comment by zjy365** [comment]
> Just wrap your content in empty tags like so: <>your content...</>

Thanks, it worked for me.

**Comment by zhiqingmai** [comment]
> Just wrap your content in empty tags like so: <>your content...</>

a new warning happened: Invalid prop `propsFromTrigger` supplied to `React.Fragment`. React.Fragment can only have `key` and `children` props.
